frontend/pages/policies/ManagePoliciesPage/components/AddPolicyModal/AddPolicyModal.tsx

```diff
@@ -70,7 +101,20 @@ const AddPolicyModal = ({
     teamId,
   ]);
 
-  const policiesAvailable = DEFAULT_POLICIES.map((policy: IPolicyNew) => {
+  const onPlatformFilterChange = (platformSelected: string) => {
```

Comment:
This is agreat use case for our `SelectedPlatform` type

Reply:
ahh thank you!!